packages/mdc-segmented-button/_index.scss

```diff
@@ -20,4 +20,109 @@
 // THE SOFTWARE.
 //
 
-// TODO: Add styles.
+@use "@material/button";
```

Comment:
`_index.scss` file should not contain any style rules. You can define styles in `/segmented-button/_segmented-button.scss` and `/segment/_segment.scss`,